includes/Checker/Checks/Plugin_Repo/File_Type_Check.php

```diff
@@ -260,6 +260,15 @@ protected function look_for_application_files( Check_Result $result, array $file
 	protected function look_for_badly_named_files( Check_Result $result, array $files ) {
 		$conflict_chars = '!@#$%^&*()+=[]{};:"\'<>,?/\\|`~';
 
+		$plugin_path = $result->plugin()->path();
+
+		$files = array_map(
+			function ( $file ) use ( $plugin_path ) {
+				return substr( $file, strlen( $plugin_path ) );
```

Comment:
In other places like `Find_Readme` we do `$file = str_replace( $plugin_path, '', $file );`